### DISCUSSION TITLE
Small clarification needed WRT importing images when using the embedded registry feature

### DISCUSSION THREAD

**Discussion by matthewadams** [k3s-io/k3s#9518] [Q&A] [answered]
We're looking to pivot from using a private registry to using the new embedded registry feature (#8977).  [The docs](https://docs.k3s.io/installation/registry-mirror#enabling-the-distributed-oci-registry-mirror) say

> When enabled at a cluster level, all nodes will host a local OCI registry on port 6443, and publish a list of available images via a peer to peer network on port 5001. Any image available in the containerd image store on any node, can be pulled by other cluster members without access to an external registry.

What is the correct `docker` command we should use to import images into the embedded registry after k3s is already up & running?

**Comment by brandond** [comment, marked as answer]
This is covered in the docs:
https://docs.k3s.io/installation/registry-mirror#pushing-images

**Comment by matthewadams** [comment]
Jeez, I hadn't gotten that far down in that section yet... 🙄

**Comment by matthewadams** [comment]
@brandond, follow up questions, mostly for a `ctr` noob.  Does k3s ship with its own `ctr` executable preconfigured to point at the embedded registry?  If so, what's its path?  If not, how do I point my own `ctr` at the embedded registry?  Is there some particular value for the `--address` argument that differs from the default of `/run/containerd/containerd.sock`?

**Reply by brandond** [reply]
we do include ctr with k3s, and it is configured to use the k3s default container runtime socket path. It should be in your path if you use the install script.

**Reply by matthewadams** [reply]
Thanks! 🙏🏼
